### PR TITLE
Enable building and installation for Fedora

### PR DESCRIPTION
**Pull request by juliogonzalez**
Port of changes at https://build.opensuse.org/package/view_file/home:ethanb11:Fedora_Uyuni:Master:Fedora30-Uyuni-Client-Tools/salt/salt.spec?expand=1 (https://github.com/openSUSE/salt-packaging/pull/28), for salt 3000

Reference: https://github.com/uyuni-project/uyuni/issues/2085

### CODE REVIEW

File: salt/salt.spec [hunk 1]
```diff
@@ -330,8 +330,8 @@ Requires:       iproute
 %endif
 
 %if %{with systemd}
-BuildRequires:  pkgconfig(systemd)
```
Comment (juliogonzalez) [inline salt/salt.spec]: Same question as for the other PR

Comment (EthanB11) [reply salt/salt.spec]: See comment on other PR I didn't make that change. It was already there in the OBS branch I used. https://build.opensuse.org/package/view_file/systemsmanagement:Uyuni:Master:CentOS8-Uyuni-Client-Tools/salt/salt.spec?expand=1

**Review by EthanB11 — APPROVED** [review]
Looks good.

**Comment by meaksh** [comment]
I've already pushed this to `3000` branch at https://github.com/openSUSE/salt-packaging/commit/c96fc684a059121ecc7baf013768b1b611777d73

Closing PR. Thanks!

**Pull request closed without merging**